resources/resource_transformers/tocss/dartsass/integration_test.go

```diff
@@ -68,22 +68,24 @@ $moolor: #fff;
 moo {
 	color: $moolor;
 }
--- assets/scss/another.css --
-
+-- node_modules/foo/another.css --
+another {
+	color: #000;
+}
 -- assets/scss/main.scss --
 @import "moo";
 @import "regular.css";
 @import "moo";
-@import "another.css";
+@import "another";
 
 /* foo */
 -- assets/scss/regular.css --
 
 -- config.toml --
 -- layouts/index.html --
-{{ $r := resources.Get "scss/main.scss" |  toCSS (dict "transpiler" "dartsass")  }}
+{{ $cssOpts := (dict "includePaths" (slice "node_modules/foo") "transpiler" "dartsass") }}
```

Comment:
The original issue, as I understand it, was about importing CSS files from `/assets`; with that I don't understand why you move the CSS to `node_modules`?

Reply:
> The original issue, as I understand it, was about importing CSS files from /assets

The OP incorrectly commented on what was working vs. what was not. I discovered their error when I began testing this, but did not ask the OP to verify their comments until yesterday. They confirmed that the original comments were reversed, but the issue title is correct.

We have no problem importing a CSS file is the assets directory at compile time. But when you import a CSS file from a mounted directory (either through `module.mounts` or the `includePaths` slice), the result is a plain import.

I would be happy to clarify the test if necessary, but I was trying to minimize the `diff` for you.

Reply:
>I would be happy to clarify the test if necessary, but I was trying to minimize the diff for you.

To minimise the diff would be dropping the includePaths change, which is what confuses me -- even after rereading the original issue. I think I understand the issue, and if I understand it correctly that should be possible to reproduce by just importing a CSS file without using the extension when the file is imported into /assets in a mount (which lives outside of what includePaths is all about).